lib/commands/import.js

```diff
@@ -219,6 +221,28 @@ const serviceMapping = {
     const finishFn = () => log.info(service.summaryReport(options.from))
     return [ options, '*', processFn, finishFn ]
   },
+  [allowedTypes.customObject]: (options) => {
+    const service = new CustomObjectImporter(Object.assign({}, options.config, {
+      apiConfig: {
+        host: options.authHost,
+        projectKey: options.credentials.project_key,
+        credentials: {
+          clientId: options.credentials.client_id,
+          clientSecret: options.credentials.client_secret,
+        },
+        apiUrl: options.host,
+        protocol: options.protocol,
+        oauth_protocol: options.authProtocol,
+      },
+      accessToken: options.accessToken,
+      batchSize: options.batch,
+      continueOnProblems: options.continueOnProblems,
```

Comment:
Hello @daern91 .. do we have somewhere this parameter? `continueOnProblems`
It is not configured here https://github.com/sphereio/sphere-node-cli/blob/fdbfcb96ffb263fd5de7d986c9bfe168dc9de4c8/lib/commands/import.js#L252